docs/share_usage.md

```diff
@@ -295,31 +309,288 @@ happens asynchronously on a child thread.
+
+**Note:** In order to use file share feature, it is mandatory to specify
+'filePersona' option with empty string as value.
+
+```yaml
+$ sudo kubectl create -f - << EOF
+---
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: sc1
+provisioner: hpe.com/hpe
+parameters:
+  filePersona: ""
+EOF
+```
+
+#### Supported StorageClass parameters<a name="sc_parameters"></a>
+
+| StorageClass Options | Type    | Parameters                                 | Example                          |
+|----------------------|---------|--------------------------------------------|----------------------------------|
+| size                 | integer | Size of share in GiB                       | size: "10"                       |
+| cpg                  | String  | Name of the CPG                                   | cpg: SomeCpg             |
+| fpg                  | String  | Existing FPG name including legacy FPG     | fpg: SomeFpg           |
+| fsMode               | String | Unix style permissions or ACL string                                | fsMode: "A:fd:rwa,A:g:rwaxdnNcCoy,A:fdS:DtnNcy"              |
+| fsOwner              | String | User Id and Group Id that should own the mounted directory      | fsOwner: "1000:1000"         
+
+### Persistent Volume Claim Example<a name="pvc"></a>
+
+Now let’s create a claim **PersistentVolumeClaim** (**PVC**). Here we specify the **PVC** name **pvc1** and reference the **StorageClass "sc1"** that we created in the previous step.
+
+```yaml
+$ sudo kubectl create -f - << EOF
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: pvc1
+spec:
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: 20Gi
+  storageClassName: sc1
+EOF
+```
+
+At this point, after creating the **SC** and **PVC** definitions, the volume hasn’t been created yet. The actual volume gets created on-the-fly during the pod deployment and volume mount phase.
+
+### Pod Example<a name="pod"></a>
+
+So, let’s create a **pod "pod1"** using the **nginx** container along with some persistent storage:
+
+```yaml
+$ sudo kubectl create -f - << EOF
+---
+kind: Pod   
+apiVersion: v1
+metadata:
+  name: pod1
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+    volumeMounts:
+    - name: export
+      mountPath: /export
+  restartPolicy: Always
+  volumes:
+  - name: export
+    persistentVolumeClaim:
+      claimName: pvc1
+EOF
+```
+
+When the pod gets created and a mount request is made, the volume is now available and can be seen using the following command:
+
+```
+$ docker volume ls
+DRIVER   VOLUME NAME
+hpe      export
+```
+On the Kubernetes/OpenShift side, it should now look something like this:
+```
+$ kubectl get pv,pvc,pod -o wide
+NAME       CAPACITY   ACCESSMODES   RECLAIMPOLICY   STATUS    CLAIM            STORAGECLASS   REASON   AGE
+pv/pv1     20Gi       RWO           Retain          Bound     default/pvc1                             11m
```

Comment:
ACCESSMODES value needs to be chaned.
It is better to add PV example yml

Reply:
I do see below example of PV

Reply:
@imran-ansari ACCESSMODES value needs to be changed.

Reply:
> @imran-ansari ACCESSMODES value needs to be changed.

Done!